starters/gatsby-starter-catalyst-bery/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -69,9 +70,10 @@ export default merge(BaseTheme, {
   },
   styles: {
     a: {
-      color: "link",
-      textDecoration: "underline",
+      color: "primary",
+      textDecoration: "none",
       ":hover": {
+        color: "secondary",
         textDecoration: "none",
```

Comment:
I would like the links to remain underlined as well - also for a11y reasons. Makes them stand out more against the surrounding text.